<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="DrawingCanvas" name="drawing_canvas" native="true">
      <widget class="QPushButton" name="convex_hull_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>56</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Convex Hull</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_all_button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>56</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingCanvas</class>
   <extends>QWidget</extends>
   <header>drawing_canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
